velox/core/QueryCtx.h

```diff
@@ -390,6 +390,12 @@ class ExecCtx {
     return optimizationParams_;
   }
 
+  void testingPeelingEnabled(bool enabled) {
```

Comment:
Do we need this? Why can't we just specify the desired config? CC: @bikramSingh91

Reply:
ExecCtx used by the test facility is a member of base class FunctionBaseTest.h